src/app/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import styles from './ProgressBar.module.css';
+import './custom.css';
```

Comment:
This is needed for the loader?

Reply:
Yes, I need the custom.css to style the progress bar and the styles.module for the div around it. The size of the chart is determined by its parent div.

Reply:
Okay, feel free to merge